src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 3 && days < 7) {
+    return 40 * days - 20;
+  } else if (days >= 7) {
+    return 40 * days - 50;
+  }
+
+  return 40 * days;
 }
```

Comment:
1) try to avoid using `magic` number. Create variable for your numeric values
2) `else if` is redundant since you use `early` return. It's enough to do like this
```
if (condition1) {
 return value1
}

if (condition2) {
 return value2
}

return value3
```